src/test/java/io/xpring/xrpl/FakeXRPProtobufs.java

```diff
@@ -0,0 +1,58 @@
+package io.xpring.xrpl;
+import com.google.protobuf.ByteString;
+import org.xrpl.rpc.v1.*;
+
+import java.io.UnsupportedEncodingException;
+
+/** Common set of fake objects - protobuf and native Java conversions - for testing */
+public class FakeXRPProtobufs {
+    // primitive test values
+    static String testCurrencyName = "currencyName";
+    static ByteString testCurrencyCode;
+
+    // TODO(amiecorso): not sure if this is the correct way to construct a ByteString...
+    // trying to create equivalent of new Uint8Array([1, 2, 3]) in JS
+    static {
+        try {
+            testCurrencyCode = ByteString.copyFrom("123", "Utf8");
```

Comment:
This seems like it should work....

If you were to go with a `byte[]`, you could do `"123".getBytes("UTF8")`, but that probably does the same thing.

Reply:
the testCurrencyCode still has to be a ByteArray because this is what's actually feeding construction of a real protobuf object, which wants to eat googly things